pkg/meta/tkv.go

```diff
@@ -486,8 +486,8 @@ func (m *kvMeta) doCleanStaleSession(sid uint64) {
 	}
 }
 
-func (m *kvMeta) doFindStaleSessions(ts int64) ([]uint64, error) {
-	vals, err := m.scanValues(m.fmtKey("SH"), func(k, v []byte) bool {
+func (m *kvMeta) doFindStaleSessions(ts int64, limit int) ([]uint64, error) {
+	vals, err := m.scanValues(m.fmtKey("SH"), limit, func(k, v []byte) bool {
```

Comment:
When the first 1000 sessions are not stale, we will get nothing. So we should limit the stale session, not scanned sessions.

Reply:
It limits the number of results, i.e. after the filter.